app/settlement.js

```diff
@@ -2,8 +2,9 @@ const db = require('./data')
 
 const updateSettlementStatus = async (returnData) => {
   if (returnData.settled) {
-    await db.completedPaymentRequest.update({ settled: returnData.settlementDate }, { where: { invoiceNumber: returnData.invoiceNumber } })
+    await db.completedPaymentRequest.update({ lastSettlement: returnData.settlementDate, settledValue: db.Sequelize.literal(`COALESCE("settledValue", 0) + ${returnData.value}`) }, { where: { invoiceNumber: returnData.invoiceNumber } })
```

Comment:
Just confirming, `settled` is still returned from DAX we are just adding the extra field `value` to that return file